_gitbook/conventions/coding_style.md

```diff
@@ -0,0 +1,140 @@
+# Coding Style
+
+This style is used in the standard library. You can use it in your own project to make it familiar to other developers.
+
+## Naming
+
+__Type names__ are camelcased. For example:
+
+```crystal
+class ParseError < Exception
+end
+
+module HTTP
+  class RequestHandler
+  end
+end
+
+alias NumericValue = Int32 | Int64 | Float32 | Float64
+
+lib LibYAML
+end
+
+struct TagDirective
+end
+
+enum Time::DayOfWeek
+end
+```
+
+__Method names__ are underscore-cased. For example:
+
+```crystal
+class Person
+  def first_name
+  end
+
+  def date_of_birth
+  end
+
+  def homepage_url
+  end
+end
+```
+
+__Variable names__ are underscore-cased. For example:
+
+```crystal
+$global_greeting = "Hello world"
+
+class Greeting
+  @@default_greeting = "Hello world"
+
+  def initialize(@custom_greeting=nil)
+  end
+
+  def print_greeting
+    greeting = @custom_greeting || @@default_greeting
+    puts greeting
+  end
+end
+```
+
+__Constants__ are screaming-cased. For example:
+
+```crystal
+LUCKY_NUMBERS = [3, 7, 11]
+DOCUMENTATION_URL = "http://crystal-lang.org/docs"
+```
+
+### Acronyms
+
+In class names, acronyms are _all-uppercase_. For example, `HTTP`, and `LibXML`.
+
+In method names, acronyms are _all-lowercase_.  For example `#from_json`,  `#to_io`.
+
+### Libs
+
+`Lib` names are prefixed with `Lib`. For example: `LibC`, `LibEvent2`.
```

Comment:
Again, "For example" sounds kind of weird here, IMO. You could just use the colon.